### DISCUSSION TITLE
spliced aware aligner for RNA-seq data analysis

### DISCUSSION THREAD

**Discussion by santataRU** [GuyTeichman/RNAlysis#38] [General]
Hi, All,

I intend to use RNAlysis to perform differential gene expression analysis with DEseq2 between two RNA-seq datasets (e.g., control vs treated).

Is there a splice-aware aligner for RNA-seq data analysis (e.g., STAR aligner) in the RNAlysis package? I know Bowtie2 is included in the RNAlysis package, but it may not be optimal for RNA-seq data.

Thanks,

Xiao

**Comment by GuyTeichman** [comment]
Hi Xiao,
I would recommend kallisto - it is a pseudoaligner that generates counts at the transcript level, but can also provide gene-level counts while maintaining splice-aware assignment. 
You can read more about it here:
https://github.com/pachterlab/kallisto

Feel free to follow up if you have any other questions!

Good luck with your analysis,
Guy.
